hack/scripts/sync-prow.sh

```diff
@@ -0,0 +1,39 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+check_deps() {
+  if ! which git-filter-repo > /dev/null 2>&1; then
```

Comment:
FYI Bash comes with the `command` builtin command which is better than running `which` as that spawns a subprocess. So this could have been just `if ! command -v git-filter-repo; then`.

Reply:
I had suggested using 'type' also builtin. I had not heard of command. Seems like with -v it's very similar to type?

Reply:
They are similar. I think `command -v` is basically the same as `type -P` which forces type to consider only those things in the PATH (to disregard any available shell aliases, for example).

I also thought of `type` sometime after writing my comment, but probably `command` is more obvious to folks not familiar with Bash, simply because "type" is such a generic word as to have no meaning without context. At least "command" gets us *some* context about what it does.